docs/README.md

```diff
@@ -48,6 +48,40 @@ The former is considered the safer option as it prevents accidental overwriting
 (as `REGRESSION_GOLD_STANDARD_SYNC` is `ON` by default).
 In the pull request of the suggested changes we will then update the official gold standard release file and appropriate hash prior to merging.
 
+#### Usage in downstream codes
+
+The easiest/recommended way to reuse the `Parthenon` regression test infrastructure in downstream codes is to adapt a similar directory structure.
+The following steps have been tested to work when Parthenon is built from source in the downstream project:
+
+1. Add the following to the downstream root `CMakeLists.txt` after `Parthenon` has been included:
+```
+include(CTest)
+add_subdirectory(tst/regression)
+```
+2. Create the following directories and files in the project folder (for an example `my_first_test` test):
+```
+tst/
+    regression/
+        CMakeLists.txt
+        test_suites/
+            __init__.py  # <-- empty file
+            my_first_test/
+                __init.py__  # <-- empty file
+                my_first_test.py
+```
+3. Contents of `tst/regression/CMakeLists.txt`
+```
+
+# import Parthenon setup_test_serial and setup_test_parallel
+include(${PROJECT_SOURCE_DIR}/external/parthenon/cmake/TestSetup.cmake)
```

Comment:
Might want to point out somewhere above that the user also needs to have the parthenon submodule in external folder in the root project source directory. I know it may seem obvious... but it would probably make it more clear.

Reply:
Based on your documentation I would add something like. For easiest use the downstream author should have a directory structure with the following form:

```
project_foo/external/parthenon - (parthenon submodule)
project_foo/bin - (location of downstream binaries/apps/executables)
```

Maybe add something like this to your existing description.

Reply:
I added the info to the parthenon directory location and made the path to the bin in the example absolute.